README.md

```diff
@@ -1,8 +1,71 @@
-# onix-docs
-Documentation, technical reference manual, and examples for the Bonsai ONI packages
+# Onix1 Bonsai Docs
 
-# Building docs locally
+Documentation for the Onix1 Bonsai packages.
 
-To build everything, including the Bonsai workflows, and view locally, run
+These docs are built on [docfx](https://dotnet.github.io/docfx/index.html).
+
+## Initialize the Documentation for Building Locally
+
+These actions only need to be performed once when setting up the repo locally.
+
+Download [dotnet](https://dotnet.microsoft.com/en-us/download).
+
+To clone the remote repository into the active directory, run:
+
+`git clone https://github.com/open-ephys/onix1-bonsai-docs`
+`cd onix1-bonsai-docs`
+
+To pulls in the latest files from the submodules according to the commit that the submodules point to, run:
+
+`git submodule update --recursive --init`
+
+To configure dotnet, run:
+
+`dotnet tool restore --configfile .\.bonsai\NuGet.config `
+
+In particular, restoring the config file configures docfx version and docfx companion tools such as [DocLinkChecker](https://github.com/Ellerbach/docfx-companion-tools/tree/main/src/DocLinkChecker).
+
+To make the `docfx` command available after restoring the config file from the previous step, run:
+
+`dotnet tool restore`
+
+To set up a local Bonsai environment for automatically exporting SVGs, run: 
+
+`./.bonsai/Setup.cmd`
+
+## Build Documentation Locally
+
+To build the docs and serve locally, run:
+
+`./build.ps1 --serve`
+
+If SVGs are already exported and do not need to be updated, they don't need to be re-exported. In that case, to build the docs and serve locally more quickly, run:
+
+`dotnet docfx --serve`
+
```

Comment:
It might be useful to add in the specific commands used during the Github Actions to see build errors locally. For example, adding the following lines:

To test links in the Markdown files locally (including cross-referencing, files, and external links), run the following commands:
`dotnet DocLinkChecker -v -f .github/workflows/DocLinkChecker.config`
`.\build.ps1 --logLevel Warning --warningsAsErrors`

Additionally, once commits are pushed to Github, an Action will run that uses [lychee](https://github.com/lycheeverse/lychee) to fully test the built HTML pages. Lychee can be installed to run locally following the instructions at the linked page, or installing the appropriate [*.exe file](https://github.com/lycheeverse/lychee/releases). Once it has been installed, you can test the pages locally by running the following command from the base directory of the documentation repo (i.e., onix1-bonsai-docs):

```powershell
lychee --verbose --no-progress --base _site --exclude ^https://github\.com.*merge.* --exclude ^https://github\.com.*apiSpec.* '_site/**/*.html'
```

Reply:
Yeah I needed this last night. I will add this once I get it going. Thank you so much